pkgdb2/templates/package.html

```diff
@@ -394,13 +394,17 @@ <h4 title="`approveacls` ACL">Package Administrator(s)</h4>
               </form>
             </li>
           {% endif %}
-          {% if g.fas_user.username in admins or is_admin %}
+          {% if g.fas_user %}
             <li>
               <a href="{{ url_for('.update_acl',package=package.name,
                        update_acl='approveacls') }}"
-                title="Manage commit ACLs">
+                title="Manage approveacls ACLs">
                 <button class="bold pkg_button_blue inline wheel">
-                  Manage the admins
+                {% if g.fas_user.username in admins or is_admin%}
+                Manage the admins
+                {% else %}
+                Request Approveacls
```

Comment:
Will this really just work if you are not an admin?  Hitting the same link will request the approveacls acls for you?